pkg/flags/ipaddressport.go

```diff
@@ -32,22 +32,26 @@ type IPAddressPort struct {
 func (a *IPAddressPort) Set(arg string) error {
 	arg = strings.TrimSpace(arg)
 
-	parts := strings.SplitN(arg, ":", 2)
-	if len(parts) != 2 {
-		return errors.New("bad format in address specification")
+	host, port, err := net.SplitHostPort(arg)
```

Comment:
nit: portStr may be better, considering port sounds like a int and we do parse it into a int.